wordpress.org/public_html/wp-content/plugins/wporg-gp-plugin-directory/inc/sync/class-translation-sync.php

```diff
@@ -135,11 +139,14 @@ public function sync_plugin_translations_on_commit( $args ) {
 	public function queue_translation_for_sync( $translation ) {
 		global $wpdb;
 
-		// Only propagate current translations without warnings.
-		if ( 'current' !== $translation->status || ! empty( $translation->warnings ) ) {
+		$allowed_statuses = array( 'current', 'rejected', 'changesrequested', 'fuzzy' );
+		// Do not propagate waiting translations and other translations with warnings.
+		if ( ! in_array( $translation->status, $allowed_statuses ) || ! empty( $translation->warnings ) ) {
 			return;
 		}
 
+		do_action( 'translation_synced', $translation );
```

Comment:
```suggestion
		do_action( 'wporg_translate_translation_synced', $translation );
```

This should get a prefix so it's clear that this is a custom hook.